buildkite/scripts/bench/run.sh

```diff
@@ -6,18 +6,21 @@ set -eox pipefail
 
 YELLOW_THRESHOLD="0.1"
 RED_THRESHOLD="0.3"
-EXTRA_ARGS=""
+EXTRA_ARGS="${EXTRA_ARGS:-}"
 
 source buildkite/scripts/bench/install.sh
 
-MAINLINE_BRANCHES="-m develop -m compatible -m master -m dkijania/enhance_benchmarks"
+MAINLINE_BRANCHES="-m develop -m compatible -m master -m dkijania/bench_for_ledger_test"
```

Comment:
This is a way to test upload of benchmark

Reply:
Will we remove it in a follow-up PR?